yarn-project/pxe/src/note_processor/utils/produce_note_daos_for_key.ts

```diff
@@ -128,6 +128,9 @@ async function handlePartialNote<T>(
             daoConstructor,
           );
         } catch (e) {
```

Comment:
Sneaked this in based on comment in https://github.com/AztecProtocol/aztec-packages/pull/8725